quickjs.h

```diff
@@ -327,7 +350,7 @@ JS_EXTERN JSRuntime *JS_NewRuntime(void);
 JS_EXTERN void JS_SetRuntimeInfo(JSRuntime *rt, const char *info);
 /* use 0 to disable memory limit */
 JS_EXTERN void JS_SetMemoryLimit(JSRuntime *rt, size_t limit);
-JS_EXTERN void JS_SetDumpFlags(JSRuntime *rt, uint64_t flags);
+JS_EXTERN int JS_SetDumpFlags(JSRuntime *rt, uint64_t flags);
 JS_EXTERN size_t JS_GetGCThreshold(JSRuntime *rt);
```

Comment:
Would you also add an API to get the current set of dump flags?
```
JS_EXTERN uint64_t JS_GetDumpFlags(JSRuntime *rt);
```

Reply:
Added. However it seems the types are incorrect. Is it correct that the API sets and gets a uint64_t while the underlying variable is defined to be a 24 bit bit-field?

Maybe this is future proofing. The original library authors also seem to do this with `JS_NewInt32` taking a context parameter when it's unused.